Puppetfile

```diff
@@ -1,6 +1,6 @@
 forge 'https://forgeapi.puppetlabs.com'
 
-mod 'ceritsc/tuned', '1.0.1'
+mod 'andylytical/tuned', git: 'https://github.com/andylytical/puppet-tuned', ref: 'topic/fix_error_on_tuned_version_fact'
```

Comment:
As discussed on bluejeans, we'd like to keep forks of upstream modules within the `lsst-it` org.